oe_theme.theme

```diff
@@ -79,11 +79,6 @@ function oe_theme_theme_suggestions_field_alter(array &$suggestions, array $vari
     // fields, while keeping all the rest markup-free.
     array_unshift($suggestions, 'field__bare');
   }
-
-  // Add suggestions for all 'Entity reference revisions' for each target type.
-  if ($variables['element']['#field_type'] === 'entity_reference_revisions') {
-    $suggestions[] = 'field__' . $variables['element']['#field_type'] . '__' . $variables['element']['#items']->getFieldDefinition()->getSetting('target_type');
-  }
```

Comment:
We can't remove this one, even if we don't use it anymore, since sites might use this suggestion to style their own things.

Reply:
fair enough.